tests/integration/targets/ndo_mcp_global_policy/tasks/main.yml

```diff
@@ -0,0 +1,300 @@
+          - query_mcp_policy_with_uuid is not changed
+          - query_all_mcp_policy is not changed
+          - query_mcp_policy_with_uuid.current == query_mcp_policy_with_name.current
+          - query_mcp_policy_with_name.current.name == "ansible_mcp_global_policy_new"
+          - query_mcp_policy_with_name.current.key == "cisco"
+          - query_mcp_policy_with_name.current.description == "A Global MCP Policy"
+          - query_mcp_policy_with_name.current.adminState == "enabled"
+          - query_mcp_policy_with_name.current.enablePduPerVlan == true
+          - query_mcp_policy_with_name.current.protectPortDisable == false
+          - query_mcp_policy_with_name.current.loopDetectMultFactor == 4
+          - query_mcp_policy_with_name.current.txFreq == 3
+          - query_mcp_policy_with_name.current.txFreqMsec == 10
+          - query_mcp_policy_with_name.current.uuid is defined
+          - query_all_mcp_policy.current | length == 1 # Can only be one
+
+    # DELETE
+    - name: Delete a MCP Global Policy with name (check mode)
+      cisco.mso.ndo_mcp_global_policy: &delete_mcp_global_policy
+        <<: *mso_info
+        template: ansible_fabric_policy_template
+        name: ansible_mcp_global_policy_changed
+        state: absent
+      check_mode: true
+      register: cm_delete_mcp_global_policy
+
+    - name: Delete a MCP Global Policy with name
+      cisco.mso.ndo_mcp_global_policy:
+        <<: *delete_mcp_global_policy
+      register: nm_delete_mcp_global_policy
+
+    - name: Delete MCP Global Policy with name again
+      cisco.mso.ndo_mcp_global_policy:
+        <<: *delete_mcp_global_policy
+      register: nm_delete_mcp_global_policy_again
+
+    - name: Assert that the MCP Global Policy was deleted with name
+      ansible.builtin.assert:
+        that:
+          - cm_delete_mcp_global_policy is changed
+          - nm_delete_mcp_global_policy is changed
+          - cm_delete_mcp_global_policy.current == nm_delete_mcp_global_policy.current == {}
+          - cm_delete_mcp_global_policy.previous.name == nm_delete_mcp_global_policy.previous.name == "ansible_mcp_global_policy_new"
+          - cm_delete_mcp_global_policy.previous.key == nm_delete_mcp_global_policy.previous.key == "cisco"
+          - cm_delete_mcp_global_policy.previous.description == nm_delete_mcp_global_policy.previous.description == "A Global MCP Policy"
+          - nm_delete_mcp_global_policy.previous.uuid is defined
+          - nm_delete_mcp_global_policy_again is not changed
+          - nm_delete_mcp_global_policy_again.previous == nm_delete_mcp_global_policy_again.current == {}
+
+    - name: Create the MCP Global Policy
+      cisco.mso.ndo_mcp_global_policy:
+        <<: *add_mcp_global_policy
+        name: ansible_mcp_global_policy_again
+        description: An MCP Global policy to delete with UUID
+      register: new_mcp_global_policy
+
+    - name: Delete the MCP Global Policy with incorrect UUID
```

Comment:
do we want to move this test to the errors section?

Reply:
Done